### ISSUE TITLE
error starting 

### ISSUE THREAD

**Issue by Denver80211** [agallego-css/ewsgui#28]
Heyo, just happen to be working on EWS scripting challenges when I saw your project and thought I would have it a go. 
Errored off the bat.  Not sure if something is taken for granted.  Running VSCode as admin:

Install-Module EWSGui -Force
start-ewsgui
 ----selected 0365----
[08:08:22][Connect-EWSService] Connected using Delegated permissions with: 
New-Object : A constructor was not found. Cannot find an appropriate constructor for type Microsoft.Exchange.WebServices.Data.OAuthCredentials.
At line:32 char:18
       return @(& $origNewObject @psBoundParameters)
                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     CategoryInfo          : ObjectNotFound: (:) [New-Object], PSArgumentException
     FullyQualifiedErrorId : CannotFindAppropriateCtor,Microsoft.PowerShell.Commands.NewObjectCommand

**Comment by agallego-css** [comment]
hey. thank you for trying this module. 

that is a known issue i'm working on.
some how when launching the Windows form (the GUI part of the tool), is failing to pop-up the authentication prompt.
the debugging error says:
**AsyncState :
AsyncWaitHandle : System.Threading.ManualResetEvent
CompletedSynchronously : False
CreationOptions : None
Exception : System.AggregateException: One or more errors occurred. (ActiveX control
'8856f961-340a-11d0-a96b-00c04fd705a2' cannot be instantiated because the current thread is
not in a single-threaded apartment.)**

but the issue occurs only on the 1st run.

if you immediately attempt to run the tool again, the tool works just fine.

also the tool doesn't work if you use Applicatoin permission flow to logon (because it doesn't need to load the authentication prompt as well).

I hope to fix this error soon, I'm also learning here about powershell and Windows Forms.

**Comment by agallego-css** [comment]
issue has been fixed in version 2.0.27
